projects/lib/src/admin/components/page-list/page-list.component.html

```diff
@@ -40,12 +36,58 @@
     <table class="table table-hover page-list-table">
       <thead>
         <tr>
-          <th>Title</th>
-          <th>Created By</th>
-          <th>Date Created</th>
-          <th>Last Updated</th>
+            <th (click)="changeSortStrategy('Title')">Title
+              <fa-layers class="fa-fw">
+                <fa-icon
+                  [icon]="faSortUp"
+                  [ngClass]="{ inactive: sortBy != 'Title' }" 
+                ></fa-icon>
+                <fa-icon
+                  [icon]="faSortDown"
+                  [ngClass]="{ inactive: sortBy != '!Title' }" 
+                ></fa-icon>
+              </fa-layers>
+            </th>
+            <th (click)="changeSortStrategy('CreatedBy')">Created By
```

Comment:
in order to simplify the logic in `changeSortStrategy()`, we'll need to update the parameter to `'Author'`